packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/backup.class.js

```diff
@@ -0,0 +1,69 @@
+import get from 'lodash/get';
+
+import {
+  BACKUP_STATE_DISABLED,
+  BACKUP_OFFER_LEGACY,
+  BACKUP_OFFER_ADVANCED,
+  BACKUP_OFFER_PREMIUM,
+} from './backup.constants';
+
+export default class Backup {
+  constructor({
+    backupDurationInReport,
+    backupOffer,
+    backupSizeInReport,
+    diskSizeInReport,
+    encryption,
+    fullDayInReport,
+    hostname,
+    mailAddress,
+    replicationZone,
+    restorePointInReport,
+    scheduleHour,
+    state,
+    vmwareVmId,
+  }) {
+    Object.assign(this, {
+      backupDurationInReport,
+      backupOffer,
+      backupSizeInReport,
+      diskSizeInReport,
+      encryption,
+      fullDayInReport,
+      hostname,
+      mailAddress,
+      replicationZone,
+      restorePointInReport,
+      scheduleHour,
+      state,
+      vmwareVmId,
+    });
+
+    this.backupSizeInReport = backupSizeInReport || false;
+    this.fullDayInReport = fullDayInReport || false;
+    this.restorePointInReport = restorePointInReport || false;
+    this.diskSizeInReport = diskSizeInReport || false;
+    this.encryption = encryption || false;
```

Comment:
Looks like those properties can contain more than one type, maybe we should only have one type for each property, it would make code easier to read I think

Reply:
These are all boolean properties returned by the API. They can also be null sometimes (which should be taken as false) and here we convert them to boolean values (null becomes false)